prac_06/languages.py

```diff
@@ -0,0 +1,12 @@
+from prac_06.programming_language import ProgrammingLanguage
+
+python = ProgrammingLanguage("Python", "Dynamic", True, 1991)
+ruby = ProgrammingLanguage("Ruby", "Dynamic", True, 1995)
+visual_basic = ProgrammingLanguage("Visual Basic", "Static", False, 1991)
+print(python)
+languages = [ProgrammingLanguage("Python", "Dynamic", True, 1991), ProgrammingLanguage("Ruby", "Dynamic", True, 1995),
```

Comment:
You've already created objects for python, ruby and visual_basic. Do you need to redeclare them, or simply add python, ruby, and visual_basic into a list?